### PR TITLE
:sparkles: add support for full text OCR extraction

### PR DESCRIPTION
**Pull request by sebastianMindee**
## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires a change to the official Guide documentation.

### CODE REVIEW

File: src/parsing/common/document.ts [hunk 1]
```diff
@@ -66,4 +72,18 @@ export class Document<T extends Inference> {
 
 ${this.inference?.toString()}`;
   }
+
+  private injectFullTextOcr(rawPrediction: StringDict) {
+    if (rawPrediction["inference"]["pages"].length < 1 || rawPrediction["inference"]["pages"][0]["extras"].length < 1 || !("full_text_ocr" in rawPrediction["inference"]["pages"][0]["extras"])) {
```
Comment (ianardee) [inline src/parsing/common/document.ts:77]: rewrite into multi-line statement

**Review by ianardee — CHANGES_REQUESTED** [review]

File: src/parsing/common/document.ts [hunk 2]
```diff
@@ -66,4 +72,18 @@ export class Document<T extends Inference> {
...
+      return;
+    }
+    const fullTextOcr = rawPrediction["inference"]["pages"].filter((e: StringDict) => "extras" in e).map((e: StringDict) => e["extras"]["full_text_ocr"]["content"]).join("\n");
```
[the remark this answers is not in the document]
Comment (ianardee) [reply src/parsing/common/document.ts:84]: try again ;-)

[the remark this answers is not in the document]
Comment (sebastianMindee) [reply src/parsing/common/document.ts:84]: lmao the lint-fix _de_-linted it...

[the remark this answers is not in the document]
Comment (sebastianMindee) [reply src/parsing/common/document.ts]: Can't I fix it along with theh eslintrc PR? feels needlessly fussy to worry about here

**Review by ianardee — APPROVED** [review]

**Pull request merged**